plugin/text_navigation/navigation_target_name.talon-list

```diff
@@ -0,0 +1,16 @@
+#todo: if we wanted to convert this
+# we'd probably want to rework the list to have hard-coded strings
+# like e.g. BRACES on the right side.
+# list: user.navigation_target_name
+# -
```

Comment:
I'm leaving a note here for discussion purposes. I don't think this should be converted as is.